backend/firebaseFunctions.js

```diff
@@ -69,20 +69,52 @@ export async function createCircle(data) {
 	data.members[`${UID}`] = true;
 	data.admin[`${UID}`] = true;
 	data.owner = UID;
+	data.codes = { adminCode: 0, publicCode: 0 };
 
+	let circles = Object.keys((await readData(`prayer_circle/circles`)) || {});
+	let allCodes = [];
+	for (let i = 0; i < circles.length; i++) {
+		let circle = circles[i];
+		let circleData =
+			(await readData(`prayer_circle/circles/${circle}/codes`)) || {};
+		allCodes.push(circleData.adminCode);
+		allCodes.push(circleData.publicCode);
+	}
+
+	while (data.codes.adminCode === 0 || data.codes.publicCode === 0) {
```

Comment:
this should be &&, because if not we might end up with one of the codes being 0 when written
```suggestion
	while (data.codes.admin === 0 && data.codes.public === 0) {
```

Reply:
I can make changes to the name, but shouldn't the symbol be 'or' instead of 'and'? If it's 'and' it will exit the loop as soon as one of the codes is changed.

Reply:
you're totally right, thanks for pointing that out!